tests/test_data.py

```diff
@@ -112,6 +113,49 @@ def test_terminate_backend(postgresql, data):
 
 def test_pg_get_active_connections(data, execute):
     assert data.pg_get_active_connections() == 1
-    execute("select pg_sleep(1)")
+    execute("select pg_sleep(2)")
     time.sleep(1)
     assert data.pg_get_active_connections() == 2
+
+
+def test_encoding(postgresql, data, execute):
+    """Test for issue #149"""
+    postgresql.set_session(autocommit=True)
+    with postgresql.cursor() as cur:
+        """plateform specific locales (,Centos, Ubuntu)"""
+        for encoding in ["fr_FR.latin1", "fr_FR.88591", "fr_FR.8859-1"]:
+            try:
+                cur.execute(
+                    f"CREATE DATABASE latin1 ENCODING 'latin1' TEMPLATE template0 LC_COLLATE '{encoding}' LC_CTYPE '{encoding}'"
+                )
+            except WrongObjectType:
+                continue
+            else:
+                break
+
+    postgresql.set_session(autocommit=False)
+    execute("CREATE TABLE tbl(s text)", dbname="latin1", commit=True)
+    execute(
+        "INSERT INTO tbl(s) VALUES ('initilialized éléphant')",
+        dbname="latin1",
+        commit=True,
+    )
+    execute("UPDATE tbl SET s = 'blocking éléphant'", dbname="latin1")
+    execute("UPDATE tbl SET s = 'waiting éléphant'", dbname="latin1", commit=True)
+    time.sleep(2)
+    running = data.pg_get_activities()
+    assert "éléphant" in running[0].query
```

Comment:
I'd suggest `assert "blocking éléphant" in running[0].query` and same below because all queries actually contain `éléphant` so it'd be easier to distinguish them.

Reply:
done in 1381d25ebd9792762cbf4903aab1e856ee752a1a